sycl/test-e2e/OneapiDeviceSelector/no_duplicate_devices.cpp

```diff
@@ -1,12 +1,9 @@
 // REQUIRES: opencl, cpu
 // RUN: %{build} -o %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR="opencl:*" %{run-unfiltered-devices} %t.out 1 &> tmp.txt
+// RUN: env ONEAPI_DEVICE_SELECTOR="opencl:*" %{run-unfiltered-devices} %t.out 1 > tmp.txt
 // RUN: cat tmp.txt | env ONEAPI_DEVICE_SELECTOR="opencl:*,cpu" %{run-unfiltered-devices} %t.out
 // RUN: cat tmp.txt | env ONEAPI_DEVICE_SELECTOR="opencl:cpu,cpu" %{run-unfiltered-devices} %t.out
 
-// https://github.com/intel/llvm/issues/15288
-// XFAIL: linux && gpu-intel-gen12
```

Comment:
According to https://github.com/intel/llvm/issues/15288 it seems to happen on all targets with a CPU. In https://github.com/intel/llvm/actions/runs/10765791450/job/29851035983 I see Arc and AMD machines failing as well.

Reply:
The problem is that URLZA added unguarded debug statements. So every run of any sycl app outputs ```ZE_LOADER_DEBUG_TRACE:Using Loader Library Path: 
ZE_LOADER_DEBUG_TRACE:Tracing Layer Library Path: libze_tracing_layer.so.1```  when running on stderr, even without setting DEBUG_TRACE.

This test failed because it uses `&>` to pipe both stderr and stdout to the FileCheck, which then chokes on the unexpected output.  This PR this test by using just `>`.  

The change to URLZA to NOT constantly output is under review, IIRC.